sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -98,7 +98,7 @@ class RadarLocationReceiver : BroadcastReceiver() {
 
     @SuppressLint("MissingPermission")
     override fun onReceive(context: Context, intent: Intent) {
-        Radar.logger.d("Broadcast Received | action = ${intent.action}")
+        Radar.logger.d("Broadcast received | action = ${intent.action}")
```

Comment:
Nit: `Received broadcast`?

Thanks for making these changes to keep things tight